.github/workflows/push-trigger.yml

```
@@ -39,3 +40,58 @@ jobs:
       ACTOR_DOCKER_HUB: ${{ secrets.ACTOR_DOCKER_HUB }}
       RELEASE_DOCKER_HUB: ${{ secrets.RELEASE_DOCKER_HUB }}
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_DEVOPS }}
+
+  get-db-upgrade-scripts:
+    runs-on: ubuntu-latest
+    env:
+      SERVICE_LOCATION: 'postgres-upgrade'
+      BUILD_ARTIFACT: 'postgres-upgrade-scripts'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)